src/editor.py

```diff
@@ -150,14 +150,14 @@ def generate_code(self,dir,thread):
             if self.line_score: code = code + self.generate_linescore(files)
             if self.scoring_plays: code = code + self.generate_scoring_plays(files)
             if self.highlights: code = code + self.generate_highlights(files)
-            if self.footer: code = code + self.generate_footer()
+            if self.footer: code = code + self.footer

```

Comment:
Instead of having the user include the `\n\n` could you just add them here and at the other spot?